source/codegen/metadata_validation.py

```diff
@@ -113,6 +113,14 @@ class RULES:
         Optional("linked_params_are_optional"): bool,
         Optional("mapped-enum"): str,
         Optional("is_optional"): bool,
+        Optional("optional"): bool,
```

Comment:
grpc-device uses `is_optional` and nidaqmx-python uses `optional`. Someone is going to mix these up.

Reply:
Sure. Renaming `optional` to `is_optional` in Scrapigen is affecting grpc-device metadata validation logic. 
@bkeryan Instead, Can we rename it to `is_optional_in_python`?

Reply:
Understood. I have renamed to `is_optional_in_python` in this PR https://github.com/ni/grpc-device-scrapigen/pull/155 and updated the metadata here as well.